examples/websocket-retry/websocket_retry.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how WebSocket client reconnects to the same backend when the client looses the connection.
```

Comment:
Edit: This BBE demonstrates how a WebSocket client reconnects to the same backend when the client loses the connection.